Chapter1/App/authConfig.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Configuration object to be passed to MSAL instance on creation. 
+ * For a full list of MSAL.js configuration parameters, visit:
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md 
+ */
+const msalConfig = {
+    auth: {
+        clientId: "Enter_the_Client_Id_Here", // This is the ONLY mandatory field that you need to supply.
+        authority: "https://login.microsoftonline.com/Enter_the_Tenant_Id_Here",
+        redirectUri: "http://localhost:3000", // You must register this URI on Azure Portal/App Registration.
+      },
+    cache: {
+        cacheLocation: "localStorage", // This configures where your cache will be stored
```

Comment:
If this is node sample, then `localStorage` is no longer applicable. Can you please check the relevant samples in msal-node github page for cache initialization?